scipy/sparse/csgraph/_tools.pyx

```diff
@@ -557,13 +557,18 @@ def construct_dist_matrix(graph,
 
     Notes
     -----
-    The predecessor matrix is of the form returned by
+    The predecessor matrix is of the form optionally returned by
     `shortest_path`.  Row i of the predecessor matrix contains
     information on the shortest paths from point i: each entry
     predecessors[i, j] gives the index of the previous node in the path from
     point i to point j.  If no path exists between point i and j, then
     predecessors[i, j] = -9999
 
+    It should be noted that `shortest_path` only returns distance matrix
+    by default. With `return_predecessors=True`, it returns a tuple with
+    distance matrix as its first element and predecessors array as second
+    element.
```

Comment:
IMO opinion we should add an entry to the `Returns` section for the distance matrix rather than adding an ad hoc section in the notes here

Reply:
I couldn't get your idea. `dist_matrix` entry is already there in `Returns` section (see above). `dist_matrix` entry is also there in documentation of `shortest_path`.

You mean to add this information to `dist_matrix` entry here in this function?

Reply:
Sorry I my mistake I did not notice this refers to `shortest_path`!